metals/src/main/scala/scala/meta/internal/metals/NoopLanguageClient.scala

```diff
@@ -39,9 +39,6 @@ abstract class NoopLanguageClient extends MetalsLanguageClient {
   override def metalsTreeViewDidChange(
       params: TreeViewDidChangeParams
   ): Unit = ()
-  // override def metalsTreeViewNodeReveal(
```

Comment:
Why is this change needed?

Reply:
As a rule of thumb, I don't like to keep a commented out code in the codebase. If the code is unused, then it can and probably should be removed.

Reply:
I agree! I thought the method was removed in this PR but I didn't see it was commented out.